src/ElectricPokemon.java

```diff
@@ -0,0 +1,51 @@
+public class ElectricPokemon extends PokemonSuper {
+
+    private String lightningColor;
+    private String habbit;
+
+    public ElectricPokemon(String name, int hp, String food, String lightningColor, String habbit) {
+        super(name, hp, food);
+        this.lightningColor = lightningColor;
+        this.habbit = habbit;
+    }
+
+    public String getLightningColor() {
+        return lightningColor;
+    }
+
+    public String getHabbit() {
+        return habbit;
+    }
+
+    public void setLightningColor(String lightningColor) {
+        this.lightningColor = lightningColor;
+    }
+
+    public void setHabbit(String habbit) {
+        this.habbit = habbit;
+    }
+
+    @Override
+    public void eats() {
+        System.out.println(getName() + " eats " + this.getFood());
+    }
+
+    @Override
+    public void speaks () {
+
```

Comment:
je mist hier een body van je methode. Technisch gezien heb je wel een body maar er staat niks in